LibVLCSharp/Shared/ISource.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace LibVLCSharp.Shared
+{
+    public interface ISource
```

Comment:
Are you sure both `IMediaSource` and `ISource` are needed? Wouldn't one interface be enough?

Reply:
You are right but we only need the ISource interface (I have background task scenario in mind, I will try to do an example, and in this case, we don't want to have a MediaPlayer in the interface).